vendor/engines/secure_rooms/app/controllers/secure_rooms/indala_numbers_controller.rb

```diff
@@ -0,0 +1,54 @@
+module SecureRooms
+
+  class IndalaNumbersController < ApplicationController
+
+    include TextHelpers::Translation
+
+    customer_tab :password
+    admin_tab     :all
+
+    layout "two_column"
+
+    before_action :init_current_facility
+    before_action :authenticate_user!
+    before_action :custom_authorize
+    before_action :check_acting_as
+
+    def initialize
+      @active_tab = "admin_users"
+      super
+    end
+
+    def edit
+      @user = User.find(params[:id])
+    end
+
+    def update
+      @user = User.find(params[:id])
+      if @user.update_attributes(update_user_params)
+        flash[:notice] = text("controllers.users.update.success")
+        redirect_to facility_user_path(current_facility, @user)
+      else
+        flash[:error] = text("controllers.users.update.error", message: @user.errors.full_messages.to_sentence)
+        render action: :edit
+      end
+    end
+
+    # During rendering, we want to use the Ability as if we were using the
+    # UsersController within which this IndalaNumber form is placed.
+    def current_ability
+      Ability.new(current_user, current_facility, UsersController.new)
```

Comment:
I'm not sure there's a better way, but this feels a little weird to me, especially the using `UsersController.new`. But just to be on the safe side, I would probably call this as `::Ability` to make sure we don't accidentally get the wrong thing at some point.